test/integration/forms/formapp.js

```diff
@@ -158,4 +162,83 @@ context('Formapp', function() {
       .get('button')
       .should('not.exist');
   });
+
+  specify('kitchen sink form', function() {
+    cy
+      .server()
+      .fixture('test/form-kitchen-sink.json').as('fxTestFormKitchenSink')
+      .route({
+        url: '/api/forms/*/definition',
+        response() {
+          return this.fxTestFormKitchenSink;
+        },
+      })
+      .as('routeFormKitchenSink')
+      .routeFormFields(fx => {
+        fx.data.attributes.storyTime = 'Once upon a time...';
+
+        return fx;
+      })
+      .visit('/formapp/1/new/1/1', { noWait: true })
+      .wait('@routeFormKitchenSink')
+      .wait('@routeFormFields');
+
+    cy
+      .get('.formio-component')
+      .as('formIOComponent')
+      .find('input[type=text]')
+      .first()
+      .type('hello')
+      .should('have.value', 'hello');
+
+    cy
+      .get('@formIOComponent')
+      .find('input[type=checkbox]')
+      .first()
+      .check()
```

Comment:
I'm not convinced this test wouldn't work if styles broke the checkbox/radio.. seems like we should try and click it?